src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationOptions.cs

```diff
@@ -2,19 +2,15 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.ComponentModel;
-using Microsoft.AspNetCore.Authentication;
-using Microsoft.AspNetCore.Authentication.Cookies;
 using Microsoft.AspNetCore.DataProtection;
 using Microsoft.AspNetCore.Http;
-using Microsoft.Extensions.Options;
 
-namespace Microsoft.AspNetCore.Builder
+namespace Microsoft.AspNetCore.Authentication.Cookies
```

Comment:
I don't think that's necessary with lambdas, as intellisense knows the type from the signature, the namespace hack was only needed for `new`